pyproject.toml

```diff
@@ -44,6 +44,7 @@ pytest-cov = "^6.0.0"
 setuptools = "^75.0.0"
 fastparquet = "^2024.5.0"
 pyperclip = "^1.9.0"
+pydantic = "^2.9.0"
```

Comment:
Disadvantage of running mypy on the builtin functions is that all dependencies must be included as dev dependencies. However, as this results in fewer bugs for the enduser, I think we should do it.